docs/developer-docs/getting-started/tokens-and-cycles.mdx

```diff
@@ -0,0 +1,153 @@
+---
+keywords: [beginner, tutorial, cycles, overview, icp tokens, using icp tokens, using cycles, getting cycles, how to get cycles, free cycles, local development with cycles, converting icp to cycles, convert icp, convert cycles]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import { GlossaryTooltip } from "/src/components/Tooltip/GlossaryTooltip";
+import TabItem from "@theme/TabItem";
+import { AdornedTabs } from "/src/components/Tabs/AdornedTabs";
+
+# Tokens and cycles
+
+<MarkdownChipRow labels={["Beginner", "Getting started", "Tutorial"]} />
+
+## Overview
+
+On the Internet Computer, there are three types of token assets: ICP tokens, cycles, and ICRC tokens. ICP and cycles are the most important for every developer to understand and use, as they are necessary for deploying and maintaining a canister.
+
+ICP tokens are important because they can be converted into cycles and used to pay for a canister's resource consumption. Cycles reflect the real cost of operations for resources such as physical hardware, rack space, energy, storage devices, and bandwidth.
+
+## ICP tokens
+
+ICP is the Internet Computer's native utility token. Its value is determined on the open market. ICP tokens play a key role in the governance and economics of the Internet Computer.
+
+There are three primary ways to use ICP tokens:
+
+- Convert ICP into [cycles](#cycles), which are required to pay for a <GlossaryTooltip>canister's</GlossaryTooltip> resources. **All dapp developers must acquire cycles to deploy their canister onto the mainnet.**
+
+- Trade ICP for other assets, such as other tokens.
+
+- Stake ICP in a [neuron](/docs/current/developer-docs/daos/nns/concepts/neurons/neuron-overview) in return for network governance participation and rewards.
+
+## Cycles
+
+The Internet Computer runs on a network of nodes owned and operated by a community of independent node providers distributed across the globe. Node providers spend money to run and maintain nodes, such that they purchase hardware and pay for the electricity and network bandwidth used by the node. The Internet Computer compensates these node providers on a monthly basis by minting and distributing rewards in the form of ICP tokens.
+
+Canisters consume the resources provided by these nodes when they are deployed on the mainnet. To make ICP sustainable, canisters are required to pay for the resources they use.
+
+Resource consumption is not paid in ICP tokens but **cycles**. As the canister is used, its cycles balance is continuously reduced. Eventually, the canister needs to be "topped up" with more cycles. The canister’s developer is responsible for topping up the canister.
```

Comment:
I think it would be good to provide a hint that anybody can top up a canister, not only the developers/controllers 🤔

Reply:
alternatively just remove the last sentence 🤔